README.md

```diff
@@ -4,12 +4,48 @@
 [![StyleCI Status](https://styleci.io/repos/62041635/shield?branch=master)](https://styleci.io/repos/62041635)
 
 
-The Bugsnag PHP PSR logger provides a standard interface to logging to Bugsnag.
+The Bugsnag PHP PSR logger is an implementation of the [Fig PSR logging standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-3-logger-interface.md) that provides a standard interface to logging to Bugsnag.
 
 
-## Documentation
+## Getting Started
 
-See our main PHP repo for all information regarding usage at https://github.com/bugsnag/bugsnag-php.
+
+### Installing
+
+Add `bugsnag/bugsnag-psr-logger` to your `composer.json`.  
+
+### Configuring
+
+This library provides a logger interface but uses the [bugsnag-php notifier library](https://github.com/bugsnag/bugsnag-php) as a base.  All configuration should be performed as described in the [official bugsnag-php documentation](https://docs.bugsnag.com/platforms/php/).
+
+### Using the Loggers
+
+The library provides two loggers, `BugsnagLogger` and `MultiLogger`.
+
+`BugsnagLogger` will automatically send a notification to Bugsnag if it receives a message with a severity higher than `info`.  This will allow you to notify of any handled exceptions through interfacing the logger directly with the framework you are using.  Ensure that the logger can communicate with the `bugsnag-php` library by passing the `client` object into it on creation.
+
+```php
+$bugsnag = Bugsnag\Client::make('YOUR-API-KEY-HERE');
+$logger = new Bugsnag\PsrLogger\BugsnagLogger($bugsnag);
+
+# Will send a notification to bugsnag
+$logger.log('error', 'An error occurred');
```

Comment:
Should use `$logger.error()` here instead, since that's the convenience interface.